lib/classes/PluginManager.js

```diff
@@ -21,27 +22,45 @@ class PluginManager {
   }
 
   autoloadServerlessAlphaPlugin() {
-    // get the path to the global node_modules dir
-    // see: https://docs.npmjs.com/files/folders
+    try {
+      const localPluginPath = path.join(process.cwd(), 'node_modules', 'serverless-alpha');
```

Comment:
@pmuens this might be a little limiting. Preferably the lookup would be done through the standard [require module algorithm](https://nodejs.org/api/modules.html#modules_all_together). A good module for that is 'resolve-from'